modern-make-handbook-en.md

```diff
@@ -0,0 +1,346 @@
+# Modern Make Handbook (v. 0.5-en)
+
+[![](https://i.imgur.com/a1MOR8T.png)](https://makefile.site)
+
+
+
+
+[TOC]
+
+
+<div style="page-break-after: always;"></div>
+## 1. Making your library of shortcuts
+
+– What does the cat say when it's hungry?
+
+– Meow!
+
+
+
+– What does the dog way when it's warning?
```

Comment:
feeling danger?

Reply:
I used "it is warning" here for brevity. A literal translation would be "What does the dog say when it wants to warn about danger". The shorter version has pretty much the same meaning, while "What does the dog say when it feels danger" is more abstract and does imply a single answer (the dog can whine, bark, roar, etc.)